tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -288,5 +290,37 @@ def testValidateLinks(self):
     if not instance.IsValidEntityInstance():
       self.fail('exception incorrectly raised')
 
+  def testGoodConnections(self):
+    parsed = instance_parser.parse_yaml(
+        os.path.join(_TESTCASE_PATH,
+                     'GOOD',
+                     'good_building_connections.yaml'))
+    parsed = dict(parsed)
+    entity_name = list(parsed.keys())[0]
+
+    entity = dict(parsed[entity_name])
+    instance = entity_instance.EntityInstance(entity,
+                                              self.universe,
+                                              parsed.keys())
+
+    if not instance.IsValidEntityInstance():
+      self.fail('exception incorrectly raised')
+
+  def testBadConnections(self):
+    parsed = instance_parser.parse_yaml(
+        os.path.join(_TESTCASE_PATH,
+                     'BAD',
+                     'bad_building_connections.yaml'))
+    parsed = dict(parsed)
+    entity_name = list(parsed.keys())[0]
+
+    entity = dict(parsed[entity_name])
+    instance = entity_instance.EntityInstance(entity,
+                                              self.universe,
+                                              parsed.keys())
+
+    if instance.IsValidEntityInstance():
```

Comment:
shouldn't we also check that the correct finding is generated? (I assume we're doing findings?)

Reply:
The instance validator isn't using findings right now. The check is less strict, and we focus on whether the console output is correct, because that's what clients will be using the validator for.